pom.xml

```diff
@@ -467,9 +467,9 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>8</source>
-          <target>8</target>
-          <release>8</release>
+          <source>11</source>
```

Comment:
Shouldn't we better update the izpack version?

Reply:
what version should we go for?

Reply:
It lift's from jdk 8 to 11, so I would use 6.0

Reply:
In that case I will update all the dependencies to their latest version and see what is the minimum required Java version for those and prepare for a major version then...